term/src/main/jni/termExec.cpp

```diff
@@ -71,8 +70,8 @@ static void android_os_Exec_setPtyUTF8Mode(JNIEnv *env, jobject clazz, jint fd,
         tios.c_iflag &= ~IUTF8;
     }
 
-    if (my_tcsetattr(fd, TCSANOW, &tios) < 0)
```

Comment:
As noted above _some_ Bionic versions redefine TCSANOW to be the same as TCSETS, but new ones don't, so your version of fix _may_ have worked for you there.